tests/data/slurm_docker_images/node/Dockerfile

```diff
@@ -1,7 +1,9 @@
 FROM rancavil/slurm-node:19.05.5-1
 
-RUN pip3 install clusterfutures==0.4
-RUN pip3 install https://github.com/fractal-analytics-platform/fractal-server/releases/download/1.0.0b1/fractal_server-1.0.0b1-py3-none-any.whl
 COPY slurm.conf /etc/slurm-llnl/
+COPY requirements.txt .
+RUN pip3 install -r requirements.txt
```

Comment:
Could we avoid an harcoded `requirements.txt` file (since that's not what we use to install this package)?

If I first extract the tar.gz file to a folder, and then use
```
pip install .[slurm]
```
it looks like pip takes care of collecting all requirements - even without the `requirements.txt` file.

Reply:
indeed it does, but it doesn't cache the building stage. By installing the requirements before the `pip install .` the whole layer is cached.

I do agree it is not elegant to have it hardcooded, and in fact it was a quick fix. Will open a whole issue dedicated to CI/docker improvements.